### ISSUE TITLE
How to suppress warning output to the console log

### ISSUE THREAD

**Issue by SuzGori** [pierreavn/angular-tabler-icons#31]
Thank you for allowing us to use your wonderful module.
I would like to know if there is a way to prevent the following message from being output when one of the icons is not loading properly.

`Tabler Icon not found:  Refer to documentation on https://github.com/pierreavn/angular-tabler-icons`

If not, would you please consider adding a feature?

**Comment by pierreavn** [comment]
Hi,

For now, there is no way to prevent this message.
I can implement this feature with options, for example:
```ts
@NgModule({
  imports: [
    TablerIconsModule.pick(icons, {
        ignoreWarnings: true   // <----
    })
  ],
  exports: [
    TablerIconsModule
  ]
})
export class IconsModule { }
```
Can you tell me if it could be great for you?

**Comment by SuzGori** [comment]
I think it is very good. Please respond when you can afford to.

**Comment by pierreavn** [comment]
I've just released the 1.68.1 version of the package.
Could you try to install it? with `yarn add angular-tabler-icons@1.68.1`

After that, you can easily disable warnings, using:

```ts
import { TablerIconsModule } from 'angular-tabler-icons';
import * as TablerIcons from 'angular-tabler-icons/icons';

@NgModule({
  imports: [
    TablerIconsModule.pick(TablerIcons, {
      // Ignore warnings, such as "Tabler Icon not found"
      ignoreWarnings: true,
    })
  ],
  exports: [
    TablerIconsModule
  ]
})
export class IconsModule { }
```

**Comment by SuzGori** [comment]
Quick response!!!!!!!!!!!!!!!!!!!!!!!!!!!
Thank you!!!!
